CONTRIBUTING.md

```diff
@@ -1,82 +1,79 @@
+
+-   Lawnchair uses title case for some English UI text while other languages do not use title case; opt for sentence case instead;
+
+-   Some English terminology may have not have commonly-used equivalents in other languages. In such cases, use short descriptive phrases—for example, the equivalent of `bottom row` for `dock`;
+
+-   Some languages (e.g. `French`) has variations of words depending if it's masculine or feminine (gender-specific); we recommend opting for gender-neutral words instead.
 
 ### Code
 
-For code, It's highly recommend that you use an Integrated Development Enviroment (IDE), know Java or preferably Kotlin and Git, which is a powerful Source Management Control for collaborating with multiple programmers.
+For code, it's highly recommended that you use an IDE (e.g. [Android Studio](https://developer.android.com/studio)), know [Java](https://www.java.com) (or preferably [Kotlin](https://kotlinlang.org/)), and [Git](https://git-scm.com/) which is a powerful source management control tool for collaborating with multiple programmers.
 
-Before cloning the repository, you must clone the repository and it submodule using the recursive argument:
+To clone the repository, you must clone the repository and its submodules using the recursive flag:
 
 ```
 git clone --recursive https://github.com/LawnchairLauncher/lawnchair.git
 ```
 
-To build the Lawnchair Launcher, select the `lawnWithQuickstepDebug` build type. Should you face errors relating to the `iconloaderlib` and `searchuilib` projects, run `git submodule update --init --recursive`.
+To build **Lawnchair Launcher**, select the `lawnWithQuickstepDebug` build type. Should you face errors relating to the `iconloaderlib` and `searchuilib` projects, run `git submodule update --init --recursive`.
 
 Here are some contribution tips to help you get started:
 
-- Always make sure that you're up-to-date with Lawnchair Launcher by setting your branch to `14-dev` to newer;
-  
-- Make sure your code is logical and well-formatted. If using Kotlin, see [“Coding conventions” in the Kotlin documentation](https://kotlinlang.org/docs/coding-conventions.html);
-  
-- [The `lawnchair` package](https://github.com/LawnchairLauncher/lawnchair/tree/14-dev/lawnchair) houses Lawnchair’s own code, whereas [the `src` package](https://github.com/LawnchairLauncher/lawnchair/tree/14-dev/src) includes a clone of the Launcher3 codebase with modifications. Generally, place new files in the former, keeping changes to the latter to a minimum
-  
+-   Always make sure that you're up-to-date with **Lawnchair Launcher** by setting your branch to `14-dev` or newer;
+
+-   Make sure your code is logical and well-formatted. If using Kotlin, see `Coding conventions` in the [Kotlin documentation](https://kotlinlang.org/docs/coding-conventions.html);
```

Comment:
**Coding conventions** referred to documentation in Kotlin, should not be in codeblock, and mostly be in quotation instead.

```suggestion
-   Make sure your code is logical and well-formatted. If using Kotlin, see [“Coding conventions” in the Kotlin documentation](https://kotlinlang.org/docs/coding-conventions.html);
```